beta/src/content/apis/react-dom/unmountComponentAtNode.md

```diff
@@ -2,11 +2,11 @@
 title: unmountComponentAtNode
 ---
 
-<Wip>
+<Pitfall>
 
-This section is incomplete, please see the old docs for [unmountComponentAtNode.](https://reactjs.org/docs/react-dom.html#unmountcomponentatnode)
+In React 18, `unmountComponentAtNode` was replaced by [`root.unmount()`](/apis/react-dom/client/createRoot#root-unmount).
 
-</Wip>
+</Pitfall>
 
 
 <Intro>
```

Comment:
Let’s shorten the intro like other pages

Reply:
Done 👍